jetty-websocket/websocket-javax-server/src/main/java/org/eclipse/jetty/websocket/javax/server/config/JavaxWebSocketServletContainerInitializer.java

```diff
@@ -102,7 +104,7 @@ public static void configure(ServletContextHandler context, Configurator configu
         // the initialization phase is over. (important for this SCI to function)
         context.getServletContext().setExtendedListenerTypes(true);
 
-        context.addEventListener(ContainerInitializer.asContextListener(new JavaxWebSocketServletContainerInitializer())
+        context.addServletContainerInitializer(new JavaxWebSocketServletContainerInitializer()
             .afterStartup((servletContext) ->
```

Comment:
This is confusing, as you are passing a lambda to be run after `onStartup()` with code from the same class?
Why can't you just inline the lambda code at the end of `onStartup()` (where in fact you call `afterStartupConsumer.accept()`)?